docs/src/components/ResponsiveTable.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+
+import {
+  Table,
+  TableCell,
+  TableCellProps,
+  TableProps,
+} from '@aws-amplify/ui-react';
+
+interface ResponsiveTableCellProps extends TableCellProps {
+  label: string;
+  children: React.ReactNode;
+}
+
+export function ResponsiveTableCell({
+  as = 'td',
+  children,
+  label,
+  className,
+  ...rest
+}: ResponsiveTableCellProps) {
+  const componentClasses = classNames('docs-responsiveTable__cell', className);
+
+  return (
+    <TableCell className={componentClasses} as={as} {...rest}>
+      <span className="docs-responsiveTable__label" aria-hidden="true">
+        {label}
+      </span>
+      <span className="docs-responsiveTable__value">{children}</span>
+    </TableCell>
+  );
+}
+
+interface ResponsiveTableProps extends TableProps {
```

Comment:
I extended this at first thinking it would be nice to inherit other TableProps but tbh I'm not sure that's really needed? Especially with what we use for the docs? The same with ResponsiveTableCell.